### ISSUE TITLE
Auto smelting

### ISSUE THREAD

**Issue by Ianaver** [SleepyTrousers/EnderCore#19]
Allow auto smelt to work WITH fortune (metals can drop multiple ingots) as well as ignore things that drop the item it self such as diamonds or coal

**Comment by Rukachan** [comment]
Having a blacklist or a whitelist for this would be desirable or else it could be exploited. (consider a mod that gives 9 iron if you smelt a block of iron)

**Comment by Ianaver** [comment]
I agree with Rukachan, however I also think that the blacklist should be able to use both IDs of the items  and the OreDictionary as it would allow for simple item/category additions and save a lot of time as well as cover a much broader spectrum of items than just using the item's ID.
